app/javascript/styles/mastodon-light/diff.scss

```diff
@@ -525,17 +525,6 @@ a.sparkline {
   }
 }
 
-@supports not selector(::-webkit-scrollbar) {
-  html {
-    scrollbar-color: rgba($action-button-color, 0.25)
-      var(--background-border-color);
-  }
-}
```

Comment:
Is this removal intended?

Reply:
@vmstan asking again about that, since I did not get an answer.

Reply:
Sorry I did not see this earlier. I will address this today.

Reply:
Fixed.